src/mono/wasm/runtime/binding_support.js

```diff
@@ -776,15 +777,22 @@ var BindingSupportLib = {
 			return this._get_js_id (mono_obj);
 		},
 
-		wasm_get_raw_obj: function (gchandle)
+		wasm_get_raw_obj: function (gchandle, add_in_flight)
```

Comment:
Let's also add a doc comment here explaining how the caller should determine whether ```add_in_flight``` needs to be ```true```

Reply:
done, thanks